ledger/acctupdates.go

```diff
@@ -1122,7 +1146,8 @@ func (au *accountUpdates) commitRound(offset uint64, dbRound basics.Round, lookb
 	au.accountsMu.Unlock()
```

Comment:
add a warning if not lockTaken? It should not happen but might serve as kind of protection for future code changes.

Reply:
yeah, I agree -- I think a check just after error checking the `Atomic` call would be a good sanity check.

Reply:
I can't see any reason not to add it.